### DISCUSSION TITLE
Host TeamPass in Azure

### DISCUSSION THREAD

**Discussion by DaveWebb2** [nilsteampassnet/TeamPass#4132] [General]
Has anybody successfully installed TeamPass in Azure, using the App Service and MySQL instance?  Is it even possible?

**Comment by DaveWebb2** [comment]
Just checking in to see if anybody has done this since I asked 8 months ago?

**Reply by DaveWebb2** [reply]
Well I had a crack at setting it up and got it done!  I've successfully created a MySQL instance with a Private EndPoint, an App Service with a Private EndPoint, and uploaded the TeamPass files to get it installed.

I've not finished setting it up yet, still working on things like LDAP and a custom domain, but pretty pleased I've built it natively in Azure without the need for a VM.

**Reply by DaveWebb2** [reply]
I have made some great progress with this!  I now have a fresh install of 3.1.4.8 running in an Azure App Service (Linux), using an Azure MySQL Flexible Server for the database, with the tasks scheduled to run under WebJobs in the App Service.  I also have oAuth working so that users can authenticate through Entra ID.

**Comment by simillo01** [comment]
Hi @DaveWebb2 , could you explain to me how you've configured the connection with EntraID?

**Reply by DaveWebb2** [reply]
Hello!

Here is my config inside TeamPass:

<img width="1625" height="790" alt="image" src="https://github.com/user-attachments/assets/2403769c-cb04-4fb0-b20d-dcdb506b20cd" />

You'll also need to create an App Registration in Entra as per the guide on the TeamPass site:

<img width="1297" height="837" alt="image" src="https://github.com/user-attachments/assets/b8f14ce7-5f0a-4eed-b550-e183acedda67" />
